src/OpenTracing/Exceptions/SpanAlreadyFinished.php

```diff
@@ -16,7 +16,11 @@
  */
 final class SpanAlreadyFinished extends DomainException
 {
-    public static function create(Span $span)
+    /**
+     * @param Span $span
+     * @return SpanAlreadyFinished
+     */
+    public static function forSpan(Span $span)
```

Comment:
There are the cases where this exception is being thrown when the span is finished:
```
- Overwriting the operation name
- Finishing the span
- Set a tag for the span
- Log data in the span
- Set baggage items for the span
```